testing/src/cypress/e2e/04-api-institution/03-instition-manage-designation.cy.ts

```diff
@@ -0,0 +1,115 @@
+import InstitutionHelperActions from "../../custom-command/institution/common-helper-functions.cy";
+import Authorization, {
+  ClientId,
+} from "../../custom-command/common/authorization";
+import CommonRegex from "../../custom-command/common/commonRegex";
+
+const institutionHelperActions = new InstitutionHelperActions();
+const commonRegexObject = new CommonRegex();
+
+const USERNAME = institutionHelperActions.getUserNameForApiTest();
+const PASSWORD = institutionHelperActions.getUserPasswordForApiTest();
+const TOKEN_URL = institutionHelperActions.getApiUrlForKeyCloakToken();
+
+describe("[Designation details] ", () => {
+  let token: string;
+
+  before(async () => {
+    const authorizer = new Authorization();
+    token = await authorizer.getAuthToken(
+      USERNAME,
+      PASSWORD,
+      ClientId.Institution,
+      TOKEN_URL
+    );
+  });
+
+  it("Verify GET call for designation-agreement endpoint", () => {
+    cy.request({
+      method: "GET",
+      url: institutionHelperActions.getApiForDesignationAgreement(),
+      followRedirect: false,
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    }).then((response) => {
+      const body = response.body;
+      expect(response.status).to.be.eq(200);
+      body.forEach((designation: JSON) => {
+        expect(designation.designationId).to.be.a("number");
+        expect(designation.designationStatus).to.be.oneOf([
+          "Approved",
+          "Declined",
+          "Pending",
+        ]);
+        expect(designation.endDate).to.match(commonRegexObject.dateTimeRegex);
+        expect(designation.startDate).to.match(commonRegexObject.dateTimeRegex);
+        expect(designation.submittedDate).to.match(
+          commonRegexObject.timeStampRegex
+        );
+      });
+    });
+  });
+
+  it("Verify GET call for designation-agreement view endpoint", () => {
+    cy.request({
+      method: "GET",
+      url: `${institutionHelperActions.getApiForDesignationAgreement()}/1`,
```

Comment:
Unless there is an exception, numbers along the code must be a const or an enum.
For instance, the "1" can be converted to a const like `DESIGNATION_AGREEMENT_ID_SOME_GOAL_OF_THE_TEST`.